sr_logging_common/logging/bag_rotate.py

```diff
@@ -15,34 +15,53 @@
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import
-import os
+from os import listdir, remove
+from os.path import getctime, join, exists
 import time
 import rospy
+import subprocess
 
 
 def remover(desired_bag_number, path):
     CONST_MAX_ACTIVE_BAGS_ALLOWED = desired_bag_number//2
```

Comment:
You can remove this if you are not using it anymore

Reply:
I am using it now, there was no way of fixing rosbags using the python package

Reply:
So sorry, I misclicked, I meant from __future__ thingy